data/is-cfg.json

```diff
@@ -11,6 +11,7 @@
 	"wifi": {
 		"AP": [
 			{
+		"active": false,
```

Comment:
please adjust spacing

Reply:
THX, the <"active": false> line must go above <AP>, change has been commited